package.json

```diff
@@ -79,8 +79,8 @@
     }
   },
   "scripts": {
-    "start": "react-scripts start",
-    "build": "react-scripts build",
+    "start": "REACT_APP_COMMIT=`git rev-parse HEAD` react-scripts start",
```

Comment:
this could be pulled out into a separate ./scripts/start.sh and ./scripts/build.sh, but it's not needed unless we start adding more dynamic env variables...